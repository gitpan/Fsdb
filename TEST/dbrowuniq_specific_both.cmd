prog='dbrowuniq'
args='-B -c event'
cmp='diff -c '
in=TEST/dbrowuniq_specific.in
