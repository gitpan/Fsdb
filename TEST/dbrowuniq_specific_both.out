#fsdb event i count
_null_getpage+128	10	6
_null_getpage+128	15	6
#  | /home/johnh/BIN/DB/dbcol event
#  | /home/johnh/BIN/DB/dbsort event
_null_getpage+4	16	6
_null_getpage+4	21	6
#   | dbrowuniq -B -c event
